### PR TITLE
Make hulks orkier

### PR DESCRIPTION
**Pull request by optimumtact**
RED WUNZ GO FASTA
:cl:oranges
balance: orks no longer shout their attacks over the radio
/:cl:

### CODE REVIEW

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]
they have to waaaaaaaagh over the radio too, otherwise this is a balance change

File: code/datums/mutations/hulk.dm [hunk 1]
```diff
@@ -45,7 +45,7 @@
 		return COMPONENT_CANCEL_ATTACK_CHAIN
 
 /datum/mutation/human/hulk/proc/scream_attack(mob/living/carbon/human/source)
-	source.say(pick(";RAAAAAAAARGH!", ";HNNNNNNNNNGGGGGGH!", ";GWAAAAAAAARRRHHH!", "NNNNNNNNGGGGGGGGHH!", ";AAAAAAARRRGH!" ), forced="hulk")
+	source.say("WAAAAAAAAAAAAAAGH!", forced="hulk")
```
Comment (san7890) [inline code/datums/mutations/hulk.dm:48]:
```suggestion
	source.say(";WAAAAAAAAAAAAAAGH!", forced="hulk")
```

they gotta yell it over common player

**Comment by optimumtact** [comment]
> they have to waaaaaaaagh over the radio too, otherwise this is a balance change

No they dont, I'm headcoder

**Comment by LemonInTheDark** [comment]
yes boss

**Review by LemonInTheDark — APPROVED** [review]

**Comment by san7890** [comment]
*sadly salutes*

**Review by san7890 — APPROVED** [review]

**Comment by DAKKA-WAAAGH** [comment]
### **BASED**

**Comment by Shadowflame909** [comment]
we become orks..but now no one can shit on hulk outside of its mechanical design

based. Ork Smash with giant wooden club. (science end-game tech)

**Comment by Colovorat** [comment]
image

**Comment by Dorsisdwarf** [comment]
I think that the radio scream is a pretty important tool for avoiding stealthy hulks smashing into places undetectably.

**Comment by Shadowflame909** [comment]
> I think that the radio scream is a pretty important tool for avoiding stealthy hulks smashing into places undetectably.

fair opinion but simple mobs can do the same thing as hulks + hulks arms now break from smashing walls + this is only being made because its the 3rd main cause of hulk seethe other then "my stun baton no worke"

It will be fine for balance!!

**Comment by drIcepop** [comment]
GREEN IZ BEST

**Comment by optimumtact** [comment]
> I think that the radio scream is a pretty important tool for avoiding stealthy hulks smashing into places undetectably.

counterpoint it's annoying

**Comment by Timberpoes** [comment]
Shit my finger slipped I didn't mean to post the changelog into the commit comment and merge it oh well nothing to be done now it's permanently in the game and will never be removed. uwu

**Comment by optimumtact** [comment]
> Shit my finger slipped I didn't mean to post the changelog into the commit comment and merge it oh well nothing to be done now it's permanently in the game and will never be removed. uwu

changelogs come from the PR body though, not the commit log

**Comment by optimumtact** [comment]
also I just noticed I put orks instead of hulk

**Comment by LemonInTheDark** [comment]
bad news besties, you failed to place a space after 🆑, so this change will go totally undocumented in the changelog

**Pull request merged**